src/esp/gfx/GenericDrawable.cpp

```diff
@@ -147,10 +150,16 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
     shader_->bindNormalTexture(*(materialData_->normalTexture));
   }
 
-  shader_->draw(mesh_);
+  shader_->draw(getMesh());
 }
 
 void GenericDrawable::updateShader() {
+  // Block the function if there is no renderer. Otherwise this will segfault.
+  // We don't have use for a shader anyways if we are not rendering anything.
+  if (!isRendererCreated) {
```

Comment:
Not just silently return, just let if fail. For example, CORRADE_INTERNAL_ASSERT. 

On big, critical functions, such as draw(...) I will put CORRADE_ASSERT there and for small, helper functions, I would put CORRADE_INTERNAL_ASSERT